build.gradle

```diff
@@ -134,6 +134,7 @@ dependencies {
     implementation "com.fasterxml.jackson.core:jackson-databind:2.11.3"
     implementation "com.konghq:unirest-java:3.11.02"
     implementation "io.github.jamsesso:json-logic-java:1.0.5"
```

Comment:
Could we avoid adding Guava as a dependency. Guava is used in a lot of projects and we might force customer into a lot of pain when dealing with this dependency. We have experienced this when using Hadoop, Cassandra etc

Reply:
@artur-ciocanu Thanks for the feedback.  I have removed the dependency on guava.